cdap-examples/LogAnalysis/src/test/java/co/cask/cdap/examples/loganalysis/LogAnalysisAppTest.java

```diff
@@ -57,7 +50,7 @@
   private static final String TOTAL_HITS_VALUE = "2";
   private static final String TOTAL_RESPONSE_VALUE = "2";
   private static final String RESPONSE_CODE = "200";
-  private static final String TPFS_RESULT = "127.0.0.1:2";
+  private static final String TPFS_RESULT = "127.0.1.1:1\n127.0.0.1:2\n";
```

Comment:
No you can't, there is no guarantee. It is unlikely, but the next version of Hadoop might order them differently. So the clean way of doing this would be to return a Map in Json format, then the test case can parse it as a Map and to a map.equals with en expected result.